index.html

```diff
@@ -65,9 +66,9 @@ <h1 id="portfolio">portfolio</h1>
             </article>
         </section>
     </main>
-    <footer id="contact">
+    <footer id="contact" class="footer-links">
```

Comment:
you can link A tags to classes as well if you wanted. Saves having both an id and class. 
You put:
<a href=".footer-links">contact</a>

rather than 
<a href="#contact">contact</a>

Reply:
opps `<a href="#contact">contact</a>`

Reply:
so that should of been 
you put: 
`<a href=".footer-links">contact</a>`

rather than: 
`<a href="#contact">contact</a>`

Reply:
cheers Jack, going to leave as is for now because I dont have use for classes yet